actioncable/test/subscription_adapter/redis_test.rb

```diff
@@ -14,3 +14,9 @@ def cable_config
     super.merge(driver: "hiredis")
   end
 end
+
+class RedisAdapterWithChannelPrefixTest < RedisAdapterTest
```

Comment:
I worked around my issue by subclassing `ActionCable::Server::Base` and making config an instance var. I made the tests an included module in the tests as I wasn't sure how to generalize it as the pg adapter *is* scoped by database. To generalize it, we *could* have all adapters support channel prefix, as while it's not *needed* by all adapters, it shouldn't hurt anything either.